Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -82,6 +85,13 @@ public final class AlertManager {
                 self?.loopDidComplete()
             }
             .store(in: &cancellables)
+
+        alertMuter.$configuration
+            .removeDuplicates()
+            .receive(on: RunLoop.main)
+            .dropFirst()
+            .sink(receiveValue: rescheduleMutedAlerts)
```

Comment:
Rescheduling only needs to happen in response to user action of enabling, or prematurely disabling, mute alert mode.  A natural expiration already does the right thing.

Reply:
Hm... no. natural expiration of the duration leaves the state in the `enabled` state. There needs to be a check after the natural duration expires to trigger the state change to `disable`. Or am I missing your point?

Reply:
I mean a natural expiration doesn't require rescheduling.  See above for my suggestion on making `enabled` a computed var, which depends on startTime.

Reply:
Yes, I agree with no need to reschedule. However the UI needs to be updated to reflect the change. I'll look into using timers for this.